script.js

```diff
@@ -0,0 +1,82 @@
+const addBtn = document.getElementById('form');
+// const addBtn = document.getElementById('add');
+// Declaring varibale for the books' container
+const listContainer = document.querySelector('.list_container');
+const booksList = JSON.parse(localStorage.getItem('books')) || [];
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+    // localStorage.setItem('books', JSON.stringify(booksList));
+  }
+
+  static addBook(book) {
+    const books = Book.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static removeBook(id) {
+    const books = Book.getBooks();
+    books.forEach((book, index) => {
+      if (book.id === parseInt(id, 10)) {
+        books.splice(index, 1);
+      }
+    });
```

Comment:
- [ ] when the book is removed from the list it is not removed from the local storage even after fereshing the page, kindly fix this bug

Reply:
Sir @Munsa1 , 

Can you please re-check? Because I am testing it, the 'set' & 'get' from the local storage work perfectly. 
![Capture](https://user-images.githubusercontent.com/102612396/197953277-967df4f0-8cad-4a88-98d9-4c2bef861013.JPG)